lectures/prob_dist.md

```diff
@@ -194,19 +194,19 @@ u.pmf(1)
 
 #### Binomial distribution
 
-Another useful (and more interesting) distribution is the **binomial distribution** on $S=\{0, \ldots, n\}$, which has PMF
+Another useful (and more interesting) distribution is the **binomial distribution** on $S=\{0, \ldots, n\}$, which has PMF:
 
 $$ 
     p(i) = \binom{n}{i} \theta^i (1-\theta)^{n-i}
 $$
 
 Here $\theta \in [0,1]$ is a parameter.
 
-The interpretation of $p(i)$ is: the number of successes in $n$ independent trials with success probability $\theta$.
+The interpretation of $p(i)$ is: the probability of $i$ successes in $n$ independent trials with success probability $\theta$.
 
-(If $\theta=0.5$, p(i) can be "how many heads in $n$ flips of a fair coin")
+(If $\theta=0.5$, p(i) can be the probability of $i$ heads in $n$ flips of a fair coin.)
```

Comment:
Let's drop the outside parentheses and change this to "For example, if $\theta = 0.5$, then $p(i)$ is the probability of $i$ heads in $n$ flips of a fair coin."

Reply:
Dear John,

Got it. I have dropped the outside parentheses and changed the sentences. 

Thank you for the suggestion!

Best,
Longye